README.md

```diff
@@ -156,6 +156,48 @@ Run all tests
 cargo test
 ```
 
+## Installation on GNU/Linux
```

Comment:
I'd rather just add a link in the README (otherwise we need to keep this in sync). We already have one link at the top. If you think that might be overlooked, I'd be okay with adding a short `Installation` section on *top* of the [Development](https://github.com/sharkdp/numbat#development) section, that just has a link to https://numbat.dev/doc/cli-installation.html

Reply:
Good idea. I have made this change in the latest commit.